sdk/communication/Azure.Communication.CallAutomation/src/Models/CustomContext.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.Communication.CallAutomation.Models
+{
+    /// <summary>
+    /// CustomContext details.
+    /// </summary>
+    public class CustomContext
+    {
+        /// <summary> Dictionary of VOIP headers. </summary>
+        public IDictionary<string, string> VoipHeaders { get; }
+
+        /// <summary> Dictionary of SIP headers. </summary>
+        public IDictionary<string, string> SipHeaders { get; }
+
+        /// <summary>
+        /// Creates a new CustomContext.
+        /// </summary>
+        public CustomContext(IDictionary<string, string> sipHeaders, IDictionary<string, string> voipHeaders)
```

Comment:
From the code, it seems that CustomerContext is instantiated by CallAutomation inside of Options, then customer would add new headers with ".Add".

But CustomContext also has public cosntructor - is there any instance where Developer would instantiate this class directly?

Reply:
change the constructor to 'internal', we create it for customer, customer do not need to initialize it manually.